test/spec/core/CanvasSpec.js

```diff
@@ -135,6 +135,23 @@ describe('Canvas', function() {
       expect(canvas._cachedViewbox).not.to.exist;
     }));
 
+
+    it('should remove planes', inject(function(canvas, elementRegistry) {
+
+      // given
+      canvas.createPlane('a');
+      canvas.createPlane('b');
+      canvas.setActivePlane('b');
+
+      // when
+      canvas._clear();
+
+      // then
+      expect(canvas.getPlane('a')).to.be.undefined;
```

Comment:
We can use the more general `not.to.exist` matcher here.

Reply:
fixed :+1: